core/cloudflow-operator/src/main/scala/cloudflow/operator/runner/AkkaRunner.scala

```diff
@@ -52,16 +52,13 @@ object AkkaRunner extends Runner[Deployment] {
       deployment.endpoint.map(endpoint ⇒ Container.Port(endpoint.containerPort, name = Name.ofContainerPort(endpoint.containerPort))).toList
     val k8sPrometheusMetricsPort = Container.Port(PrometheusConfig.PrometheusJmxExporterPort, name = Name.ofContainerPrometheusExporterPort)
 
-    val environmentVariables =
-      if (!agentPaths.contains(CloudflowApplication.PrometheusAgentKey)) {
-        List(EnvVar(JavaOptsEnvVar, ctx.akkaRunnerSettings.javaOptions))
-      } else {
+    val environmentVariables = List(EnvVar(JavaOptsEnvVar, ctx.akkaRunnerSettings.javaOptions)) ++
+      if (agentPaths.contains(CloudflowApplication.PrometheusAgentKey)) {
         List(
-          EnvVar(JavaOptsEnvVar, ctx.akkaRunnerSettings.javaOptions),
           EnvVar(PrometheusExporterPortEnvVar, PrometheusConfig.PrometheusJmxExporterPort.toString),
           EnvVar(PrometheusExporterRulesPathEnvVar, PrometheusConfig.prometheusConfigPath(Runner.ConfigMapMountPath))
         )
-      }
+      } else Nil
```

Comment:
@maasg Possibly we don't need this any longer. This was required for backwards compatibility with the single image version which we are not having anyway. But needs testing ..